src/stories/containers/cu-table/cu-table.tsx

```diff
@@ -381,14 +382,34 @@ export const CuTable = () => {
     }
     return (
       <Header>
-        <Title isLight={isLight}>Core Unit Expenses</Title>
+        <Title isLight={isLight}> Core Unit Expenses</Title>
+        <FilterButtonWrapperMobile>
+          <CustomButton
+            label="Reset Filters"
+            style={{
+              width: '114px',
+              border: 'none',
+              background: isLight ? 'none' : 'none',
```

Comment:
If in both cases the value is none, it maybe makes sense just to remove the ternary operator a set `none` instead

Reply:
You are right